File: Treasury.Domain/Model/Frminute.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Treasury.Domain.Model
{
    public partial class Frminute
    {
        public int Id { get; set; }
        public string AgendaNumber { get; set; }
        public int FrId { get; set; }
        public string MinutesLink { get; set; }
        public DateTime Timestamp { get; set; }

        public virtual FundingRequest Fr { get; set; }
    }
}

File: src/main/java/edu/wpi/sga/treasury/domain/model/FRMinute.java
package edu.wpi.sga.treasury.domain.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDateTime;

@Getter
@Setter
@Entity
@Table(name = "fr_minute")
public class FRMinute {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @OneToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "funding_request_id", nullable = false)
    private FundingRequest fundingRequest;

    @Column(name = "agenda_number", nullable = false, length = 9)
    private String agendaNumber;

    @Column(name = "minutes_link")
    private String minutesLink;

    @Column(name = "last_modified", nullable = false)
    private LocalDateTime lastModified;

}